packages/tines/data_stream/audit_logs/manifest.yml

```
@@ -0,0 +1,99 @@
+    title: Audit Logs
+    description: Collect Tines Audit Logs
+    vars:
+      - name: ssl
+        type: yaml
+        title: SSL Configuration
+        description: i.e. certificate_authorities, supported_protocols, verification_mode etc.
+        multi: false
+        required: false
+        show_user: false
+        default: |
+          #certificate_authorities:
+          #  - |
+          #    -----BEGIN CERTIFICATE-----
+          #    MIIDCjCCAfKgAwIBAgITJ706Mu2wJlKckpIvkWxEHvEyijANBgkqhkiG9w0BAQsF
+          #    ADAUMRIwEAYDVQQDDAlsb2NhbGhvc3QwIBcNMTkwNzIyMTkyOTA0WhgPMjExOTA2
+          #    MjgxOTI5MDRaMBQxEjAQBgNVBAMMCWxvY2FsaG9zdDCCASIwDQYJKoZIhvcNAQEB
+          #    BQADggEPADCCAQoCggEBANce58Y/JykI58iyOXpxGfw0/gMvF0hUQAcUrSMxEO6n
+          #    fZRA49b4OV4SwWmA3395uL2eB2NB8y8qdQ9muXUdPBWE4l9rMZ6gmfu90N5B5uEl
+          #    94NcfBfYOKi1fJQ9i7WKhTjlRkMCgBkWPkUokvBZFRt8RtF7zI77BSEorHGQCk9t
+          #    /D7BS0GJyfVEhftbWcFEAG3VRcoMhF7kUzYwp+qESoriFRYLeDWv68ZOvG7eoWnP
+          #    PsvZStEVEimjvK5NSESEQa9xWyJOmlOKXhkdymtcUd/nXnx6UTCFgnkgzSdTWV41
+          #    CI6B6aJ9svCTI2QuoIq2HxX/ix7OvW1huVmcyHVxyUECAwEAAaNTMFEwHQYDVR0O
+          #    BBYEFPwN1OceFGm9v6ux8G+DZ3TUDYxqMB8GA1UdIwQYMBaAFPwN1OceFGm9v6ux
+          #    8G+DZ3TUDYxqMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAG5D
+          #    874A4YI7YUwOVsVAdbWtgp1d0zKcPRR+r2OdSbTAV5/gcS3jgBJ3i1BN34JuDVFw
+          #    3DeJSYT3nxy2Y56lLnxDeF8CUTUtVQx3CuGkRg1ouGAHpO/6OqOhwLLorEmxi7tA
+          #    H2O8mtT0poX5AnOAhzVy7QW0D/k4WaoLyckM5hUa6RtvgvLxOwA0U+VGurCDoctu
+          #    8F4QOgTAWyh8EZIwaKCliFRSynDpv3JTUwtfZkxo6K6nce1RhCWFAsMvDZL8Dgc0
+          #    yvgJ38BRsFOtkRuAGSf6ZUwTO8JJRRIFnpUzXflAnGivK9M13D5GEQMmIl6U9Pvk
+          #    sxSmbIUfc2SGJGCJD4I=
+          #    -----END CERTIFICATE-----
+      - name: interval
+        type: text
+        title: Interval
+        description: Duration between requests to the API.
+        multi: false
+        required: true
+        show_user: false
+        default: 5m
+      - name: user_id
+        type: text
+        title: User ID
+        description: Only include logs from a matching user. Use the format user_id[]= to select multiple users, i.e. ?user_id[]=1&user_id[]=2
+        multi: false
+        required: false
+        show_user: false
+      - name: operation_name
+        type: text
+        title: Operation Name
+        description: Only include logs with a specific operation name. Use the format operation_name[]= to select multiple operation names, i.e. ?operation_name[]=StoryItemsMovement&operation_name[]=StoryItemsCreation
+        multi: false
+        required: false
+        show_user: false
+      - name: initial_lookback
+        type: text
+        title: Initial Look Back Period
+        description: Period of historical audit logs to collect when first collection request occurs
+        multi: false
+        required: true
+        show_user: false
+        default: -168h
```

```suggestion
      - name: initial_interval
        type: text
        title: Initial Interval
        description: Period of historical audit logs to collect when first collection request occurs
        multi: false
        required: true
        show_user: false
        default: 168h
```
for consistency with other packages